quinn/src/connection.rs

```diff
@@ -143,194 +152,25 @@ impl Connecting {
     ///
     /// On all non-supported platforms the local IP address will not be available,
     /// and the method will return `None`.
-    ///
-    /// Will panic if called after `poll` has returned `Ready`.
```

Comment:
This is still true, isn't it?

Reply:
Similar to above comment--I reverted all changes to this file, and so if this comment is missing it's because it's missing on main.